test/realtime/live_objects.test.js

```
@@ -895,6 +900,278 @@ define(['ably', 'shared_helper', 'chai', 'live_objects', 'live_objects_helper'],
+          action: async (ctx) => {
+            const { root, liveObjectsHelper, channel } = ctx;
+
+            // subscribe to updates on root. should only proc once CREATE ops are received for referenced objects
+            let subscribeCallbackCalledCount = 0;
+            const keyUpdated = {
+              map: false,
+              counter: false,
+            };
+            root.subscribe(({ update }) => {
+              subscribeCallbackCalledCount++;
+              Object.keys(update).forEach((x) => (keyUpdated[x] = true));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)